pipeline/terraform/stack/branch/locals.tf

```
@@ -0,0 +1,10 @@
+locals {
+  tei                         = var.toggle_tei_on ? "tei-on" : "tei-off"
```

```suggestion
  tei_suffix                         = var.toggle_tei_on ? "-tei-on" : ""
```

Then you can drop all the conditionals below.

---

hmm it doesn't quite work cause that tei flag is used to name the services with a "tei-on" or "tei-off" suffix, but for the indices we don't want a tei-off suffix, we just want no suffix. So I think the conditionals will have to stay. I think tei_suffix is a better name though